SwiftyContextMenu/ContextMenuActionTableViewCell.swift

```diff
@@ -8,25 +8,86 @@
 import UIKit
 
 class ContextMenuActionTableViewCell: UITableViewCell {
-
+    
     private var rightImageView = UIImageView(frame: CGRect(x: 0, y: 0, width: 25, height: 25))
-
+    private var lightSelectedBackgroundView: UIVisualEffectView!
+    private var darkSelectedBackgroundView: UIView!
+    private var separatorView: ContextMenuSeparatorView!
+    private var style: ContextMenuUserInterfaceStyle = .light {
+        didSet {
+            updateSelectedBackgroundView()
+            separatorView.style = style
+        }
+    }
+    
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
         backgroundColor = .clear
         rightImageView.contentMode = .scaleAspectFit
         accessoryView = rightImageView
+        
+        prepareSelectedBackgroundView()
+        
+        separatorView = ContextMenuSeparatorView(frame: CGRect(x: 0, y: 0, width: bounds.width, height: 0.33), style: self.style)
```

Comment:
can we update this code to use autolayout instead of frame layout? 
In this way we don't need to do any size updates in `layoutSubviews` ourselfs.

```
let separatorView = ContextMenuSeparatorView(frame: .zero)
separatorView. translatesAutoresizingMaskIntoConstraints = false
NSLayoutConstraint.activate([
separatorView.leading.....
])
```